### ISSUE TITLE
Need part is auto created for a grand parent need

### ISSUE THREAD

**Issue by MohSam1** [useblocks/sphinx-needs#907]
a need part is automatically getting created for its grand parent need, I think this is incorrect.
See example code below

```
.. need_grand:: Title
   :id: id_1
   :tags: xyz1

   .. need_parent:: Title
      :id: Id_2
      :tags: xyz1

      :need_part:`(child_need) title`
         :tags: xyz1

.. needflow:: Sample needflow
   :tags: xyz1

```
The above creates a needflow with a need part for need_parent and an auto generated need part of need_grand.

**Comment by PhilipPartsch** [comment]
I believe this is a doublicate of #802

**Comment by danwos** [comment]
Closed as duplicate of #802
